web_ui/frontend/app/origin/page.tsx

```diff
@@ -39,10 +39,10 @@ export default function Home() {
                     <Box sx={{backgroundColor: "#F6F6F6", borderRadius: "1rem"}} p={2}>
                         <Box minHeight={"200px"}>
                             <RateGraph
-                                rate={TimeDuration.fromString("3h")}
-                                duration={TimeDuration.fromString("7d")}
-                                resolution={TimeDuration.fromString("3h")}
-                                metric={['xrootd_server_bytes{direction="rx"}', 'xrootd_server_bytes{direction="rx"}']}
```

Comment:
Lets keep these defaults. I realize they aren't the best when you first enter but for a origin that is actually in use somewhere I think they are more helpful then just todays numbers. 

If you disagree, we should at least kick up the resolution a little bit, maybe to "20m"? 1440 data points on the graph is going to get busy.

Reply:
I agree we should think about users who run origin for days. How about I add a note to the graph to tell the user if they just started the origin and they can't see incoming data, they should modify the resolution?

Reply:
Actually, I can't run origin UI with latest main and I got the white screen with client side error that leads to `Objects are not valid as a React child (found: object with keys {Type, Value}). If you meant to render a collection of children, use an array instead.` I've rolled back the values to default but I won't be able to add the note without seeing the UI. We can talk about more in detail tomorrow.